src/components/timeline/TrackContextMenu.js

```diff
@@ -478,6 +478,7 @@ class TimelineTrackContextMenu extends PureComponent<Props> {
     // We check that it's less or equal to 1 (instead of just equal to 1)
     // because we want to also leave the item disabled when we hide the last
     // screenshot track while the menu is open.
+    // Trying to hide all types of tracks.
```

Comment:
Looks like this comment is on the wrong place. Would you mind removing or moving it?